draft-irtf-cfrg-vdaf.md

```diff
@@ -126,6 +126,16 @@ informative:
     seriesinfo: EUROCRYPT 2014
     target: https://link.springer.com/chapter/10.1007/978-3-642-55220-5_35
 
+  GSZ20:
+    title: "Guaranteed Output Delivery Comes Free in Honest Majority MPC"
+    author:
+      - ins: V. Goyal
+      - ins: Y. Song
+      - ins: C. Zhu
+    date: 2020
+    seriesinfo: CRYPTO 2020
+    target: https://link.springer.com/chapter/10.1007/978-3-030-56880-1_22
```

Comment:
@jimouris, @pratiksarkar93 let me know if you have a preferred reference for guaranteed output delivery.

Reply:
I decided against https://eprint.iacr.org/2020/300 because it's not published anywhere. However RFCs do have references to unpublished work, so we could include it if you think it's better.